app/src/main/java/org/bostwickenator/ftpuploader/FtpClient.java

```diff
@@ -9,43 +9,58 @@
 import java.io.FileInputStream;
 import java.io.IOException;
 
+import static java.lang.Integer.parseInt;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_PASSIVE;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_PASSWORD;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_PORT;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_SERVER;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_TARGET_DIR;
+import static org.bostwickenator.ftpuploader.SettingsActivity.SETTING_USERNAME;
+import static org.bostwickenator.ftpuploader.SettingsStore.settingsStore;
+
```

Comment:
Importing constants statically is frowned upon, should be used sparingly.

Reply:
Also 1990 style. ;-) It is much more readable like this if the static constant expresses enough to be understand, as it is the case here.